velox/connectors/Connector.h

```diff
@@ -304,7 +305,8 @@ class Connector {
       const std::unordered_map<
           std::string,
           std::shared_ptr<connector::ColumnHandle>>& columnHandles,
-      ConnectorQueryCtx* FOLLY_NONNULL connectorQueryCtx) = 0;
+      ConnectorQueryCtx* FOLLY_NONNULL connectorQueryCtx,
+      const std::shared_ptr<dwio::common::ReaderOptions> options = {}) = 0;
```

Comment:
Default arguments in virtual functions are not a good practice (at least according to google): https://google.github.io/styleguide/cppguide.html#Default_Arguments

Reply:
I just did a test build with your suggestion (removing the "= {}" in the Connector.h and it works!  C++ can something be really picky  LOL  Ok I will push this as a change to the PR.

Reply:
I did. :)